orchestration/dagster_orchestration/hca_manage/manage.py

```diff
@@ -240,6 +240,10 @@ def get_dataset_id(self) -> str:
         response = self.data_repo_client.enumerate_datasets(filter=self.dataset)
         return response.items[0].id  # type: ignore # data repo client has no type hints, since it's auto-generated
 
+    def enumerate_dataset(self) -> str:
+        response = f"{self.data_repo_client.enumerate_datasets(filter=self.dataset)}"
```

Comment:
❓ What happens if we pass in a bogus dataset name? Will the client throw an exception or just return nothing?

Reply:
Passing in bogus dataset names will just return an empty list.